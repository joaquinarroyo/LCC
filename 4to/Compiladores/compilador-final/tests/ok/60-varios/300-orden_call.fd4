let test : Nat =
  (let x : Nat = print "asd" 123 in (fun (y:Nat) -> x+y))
  (print "efg" 456)
