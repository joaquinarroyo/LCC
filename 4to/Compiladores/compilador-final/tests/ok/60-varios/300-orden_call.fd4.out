asd123
efg456
